Source/Core/Core/PowerPC/PowerPC.h

```diff
@@ -25,10 +25,12 @@ enum CPUCore
 {
   CORE_INTERPRETER,
   CORE_JIT64,
-  CORE_JITIL64,
-  CORE_JITARM,
+  CORE_JITIL64,  // Doesn't exist anymore
```

Comment:
It'd probably be better to just remove these and explicitly give the enum instances their respective values. i.e.

```cpp
// 2 and 3 used to be used....
// [explanation here]
enum CPUCore
{
  CORE_INTERPRETER = 0,
  CORE_JIT64 = 1,
  CORE_JITARM64 = 4,
  CORE_CACHEDINTERPRETER = 5
};
```

which removes the identifier entirely and prevents it from being accidentally used.